docs/user/concepts/rates.rst

```diff
@@ -0,0 +1,42 @@
+
+.. _disease-hazard-rates:
+
+Hazard Rates
+------------
+The hazard rate is defined first for an individual:
+
+    A *hazard rate* is the probability, per unit time, that an event
+    will happen given that it has not yet happened.
+
+For a population, the hazard rate is the sum of the hazard rates for
+all individuals in that population. For instance, the remission rate,
+as a function of age, averages over all the different times someone
+may have entered the with-condition state.
+
+The Dismod-AT compartmental model has four *Dismod-AT primary rates,*
```

Comment:
Background on Compartmental Model from Wikipedia.  Compartmental models are a technique used to simplify the mathematical modelling of infectious disease. The population is divided into compartments, with the assumption that every individual in the same compartment has the same characteristics.  Compartmental models may be used to predict properties of how a disease spreads, for example the prevalence (total number of infected) or the duration of an epidemic. Also, the model allows for understanding how different situations may affect the outcome of the epidemic, e.g., what the most efficient technique is for issuing a limited number of vaccines in a given population.

Reply:
I linked to the wikipedia article, but it hurts b/c these are things I study.